src/app/global/components/nav-desktop/nav-desktop.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+interface Item {
+  icon: string;
+  rota: string;
+  href: string;
+}
+
+@Component({
+  selector: 'app-nav-desktop',
+  templateUrl: './nav-desktop.component.html',
+  styleUrls: ['./nav-desktop.component.scss'],
+})
+export class NavDesktopComponent implements OnInit {
+  public items: Item[] = [];
+  public navFixed: boolean = true;
+  @Output() newItemEvent = new EventEmitter<boolean>();
+
+  ngOnInit(): void {
+    this.items = [
+      { icon: 'forms_add_on', rota: 'Formulário', href: '/formulario' },
+      { icon: 'group', rota: 'Pacientes', href: '/pacientes' },
+      { icon: 'folder_shared', rota: 'Tratamentos', href: '/tratamentos' },
+      { icon: 'badge', rota: 'Trabalhadores', href: '/trabalhadores' },
+      { icon: 'settings', rota: 'Configurações', href: '/configuracoes' },
+    ];
```

Comment:
faltou incluir o botão de sair, aproveita e apaga o método logout() que ta no GamemComponent . Aconselho tu usar o ConfirmDialogModule do primeng pra fazer dialog de confirmação de logout que aparece após clicar no botão de sair, assim como já existe no gamem-v1

```suggestion
    this.items = [
      { icon: 'forms_add_on', rota: 'Formulário', href: '/formulario' },
      { icon: 'group', rota: 'Pacientes', href: '/pacientes' },
      { icon: 'folder_shared', rota: 'Tratamentos', href: '/tratamentos' },
      { icon: 'badge', rota: 'Trabalhadores', href: '/trabalhadores' },
      { icon: 'settings', rota: 'Configurações', href: '/configuracoes' },
       { icon: 'icone', rota: 'Sair', href: '/' },
    ];
```